pkg/cli/uninstall/steps.go

```diff
@@ -118,29 +72,6 @@ func (u *Uninstall) deleteEverestCRDs(ctx context.Context) error {
 	return nil
 }
 
-// Older versions of Everest (> 1.3.0) were not installed using Helm.
-// So we need to render the manifests using the Helm chart and delete them using the kube client.
-// Note that the uninstallation process takes care of most resources, but cluster-scoped resources like CRDs and ClusterRoles are
-// not deleted without this step.
-func (u *Uninstall) cleanupLeftovers(ctx context.Context) error {
```

Comment:
VS [allows](https://github.com/Percona-Lab/percona-version-service/blob/main/sources/metadata/everest/1.3.0.yaml) the previous version (1.3.0) to be installed using cli 1.2.0, so without helm. Would it mean that the cleanup is still relevant?

Reply:
Not sure what you mean.. I added this clean-up thing earlier to make it possible to uninstall Everest < 1.4.0 (i.e, without helm) with CLI 1.4.0.

But now I added a [check](https://github.com/percona/everest/pull/869/files#diff-a8915bc94071568fb5c4f87eba9318927746fe4316029eb2caa21aad7b824e1dR100) such that when using CLI 1.4.0, you cannot uninstall if Everest < 1.4.0 (i.e, without helm, so that we don't have to maintain this code..)

> VS [allows](https://github.com/Percona-Lab/percona-version-service/blob/main/sources/metadata/everest/1.3.0.yaml) the previous version (1.3.0) to be installed using cli 1.2.0,

I think this is fine, as long as you uninstall previous version (1.3.0) with CLI 1.3.0, which is what the above check ensures. Am I missing something?